cms/djangoapps/bundles/views.py

```diff
@@ -45,6 +57,31 @@ class to hold user state.
     return get_user_state_kvs._kvs
 
 
+def get_xblock_handler_url(usage_id_str, handler_name, suffix, user_id):
+    """
+    Studio's implementation of a method for getting the URL to any XBlock
+    handler. The URL must be usable without any authentication (no cookie,
+    no OAuth/JWT), and may expire.
+    """
+    scheme = "https" if settings.HTTPS == "on" else "http"
```

Comment:
Should we not default this to `https`?

Reply:
This little snippet is used in various places in `edx-platform`, and the `HTTPS` setting [defaults to 'on'](https://github.com/edx/edx-platform/blob/ab23d51763e11f9f7628f57644779d82b9567d0d/cms/envs/common.py#L640) so I assume this is fine.